### PR TITLE
WHSZ surface and borehole sensors

### PR DESCRIPTION
**Pull request by mnaguit**
- sensor swap for LC 10 from Lennartz with EH-ENZ streams to TC with HH-ENZ streams
- deploy borehole sensor using TC at LC 11 with HH-12Z streams
- swap dataloggers from Q330/6 to Centaur CTR4-6S
- Centaur firmware v.4.5.20

### CODE REVIEW

**Comment by mnaguit** [comment]
Got a bit confused with elevation here, as well as the streams in LC 10 & 11 (ENZ vs 12Z). Please double check.
Also, if orientation needs to be updated, will do so once the info is available.

File: install/sensors.csv [hunk 1]
```diff
@@ -840,6 +840,8 @@ Nanometrics Inc.,Trillium Compact 120PH-2,3816,BHW,10,0,,0,0,0,0,0,0,2016-07-15T
 Nanometrics Inc.,Trillium Compact 120PH-2,3816,BHW,10,0,,0,0.7,0,0,0,0,2016-07-15T02:32:00Z,2020-07-15T01:01:00Z
 Nanometrics Inc.,Trillium Compact 120PH-2,3872,COVZ,10,335,,0,64,0,0,0,0,2016-04-07T03:30:00Z,9999-01-01T00:00:00Z
 Nanometrics Inc.,Trillium Compact 120PH-2,6637,WEL,13,0,,0,0,0,0,0,0,2021-03-30T03:00:00Z,2021-04-14T04:05:00Z
+Nanometrics Inc.,Trillium Compact 120PH-2,8743,WHSZ,10,0,,0,0,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
+Nanometrics Inc.,Trillium Compact 120PH-2,8785,WHSZ,11,0,,0,0,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/sensors.csv:844]: I feel like this should have 293 in the depth column, I will update the azimuth later when the FDSN problems are resolved!

Comment (CBurton90) [reply install/sensors.csv:844]: `Nanometrics Inc.,Trillium Compact 120PH-2,8785,WHSZ,11,0,,0,293,0,0,0,0,2021-11-03T22:57:00Z,9999-01-01T00:00:00Z`

If I've counted the columns correctly

**Review by CBurton90 — CHANGES_REQUESTED** [review]

File: install/sensors.csv (hunk 1 above)
Comment (mnaguit) [reply install/sensors.csv]: Depth modified. Thanks @CBurton90

**Review by CBurton90 — APPROVED** [review]

**Review by salichon — APPROVED** [review]
Orientation to come :) 
sensor horizontal coordinates coudl be work on for the sake of it :)

**Comment by mnaguit** [comment]
Thanks @CBurton90 & @salichon 
Are you happy with this @ozym ?

**Review by ozym — APPROVED** [review]
looks good to me, great work.

**Comment by mnaguit** [comment]
Thanks @ozym 
Will merge this PR now...if orientation needs to be updated later, will issue another one.
FYI @salichon @CBurton90

**Pull request merged**